src/Microsoft.Framework.DependencyInjection.Abstractions/ServiceProviderExtensions.cs

```diff
@@ -82,7 +75,14 @@ public static IEnumerable<T> GetRequiredServices<T>([NotNull] this IServiceProvi
         public static IEnumerable<object> GetRequiredServices([NotNull] this IServiceProvider provider, [NotNull] Type serviceType)
         {
             var genericEnumerable = typeof(IEnumerable<>).MakeGenericType(new [] { serviceType });
```

Comment:
Does the performance of this matters? We can't benefit from the generic `IEnumerable<T>` as we previously did. This could be achieved using a private method which both methods call or just duplicate code (and tests).